Makefile

```diff
@@ -3,6 +3,14 @@ GO_TEST_FLAGS = $(VERBOSE)
 DEBUG = --debug
 COVER_PROFILE = cover.out
 
+# trivialVersions=false means generate the CRD with multiple versions,
+#     eventually allowing conversion webhooks
+# allowDangerousTypes=true lets use the float64 field for clock speeds
+# crdVersions=v1 generates the v1 version of the CRD type
+# preserveUnknownFields=false causes the API server to discard "extra" data instead of storing it
```

Comment:
This is the default with the v1 CRD type.

Reply:
Yes, I wanted to be explicit. I can remove it if you think that's better.

Reply:
Sorry, I missed your comment on the other PR. It seems potentially misleading to me because it doesn't actually do anything on v1. And the option may go away at some point, since it's only there for people using v1beta1.
It would definitely be worth keeping some commentary about this on the crdVersions option though.